deploy-manage/security/self-tls.md

```diff
@@ -0,0 +1,34 @@
+---
+navigation_title: "Self-managed"
+applies_to:
+  deployment:
+    self: ga
+---
+
+% scope: TLS encryption tasks and settings after the initial setup is completed.
+# Manage TLS encryption in self-managed deployments
+
+This section provides guidance on managing TLS certificates in self-managed deployments after the initial security setup. It covers tasks such as configuring mutual authentication, renewing certificates, and customizing supported TLS versions and cipher suites.
+
+::::{note}
+If you're looking to secure a new or existing cluster by setting up TLS for the first time, refer to [](./self-setup.md), which covers both the [automatic](./self-auto-setup.md) and [manual](./self-setup.md#manual-configuration) configuration procedures.
+::::
+
+The topics in this section focus on post-setup tasks:
```

Comment:
because this last sentence kind of refers to the note, I'd just make the note into a paragraph so they can flow/be read together

```suggestion
If you're looking to secure a new or existing cluster by setting up TLS for the first time, refer to [](./self-setup.md), which covers both the [automatic](./self-auto-setup.md) and [manual](./self-setup.md#manual-configuration) configuration procedures.

The topics in this section focus on post-setup tasks:
```